### PR TITLE
terminus 1.9.0 (new formula)

### PR DESCRIPTION
**Pull request by wturnerharris**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
This got dumped when the new homebrew structure was released.

[Terminus](https://github.com/pantheon-systems/terminus) is a command-line utility for working with sites on Pantheon's infrastructure.

### CODE REVIEW

**Review by fxcoudert — DISMISSED** [review]

File: Formula/terminus.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  homepage "https://github.com/pantheon-systems/terminus"
+  url "https://github.com/pantheon-systems/terminus/archive/1.9.0.tar.gz"
+  sha256 "4e315e950affbf0c95e7b1225b80151c85ff8ae04b2aad053c2adb5c30380f93"
+
+  depends_on "Composer"
+
+  def install
+    system "composer", "install"
+    bin.install "bin/terminus" => "terminus"
+    prefix.install Dir["*"]
```
[the remark this answers is not in the document]
Comment (wturnerharris) [reply Formula/terminus.rb]: I initially tried only installing the executable, but it does require a few files. It's mainly the vendor files from composer. I'll prune it.

**Review by javian — COMMENTED** [review]
This formula used to exist in the old php tap and we decided to remove pretty much everything when we migrated over. Imo this will fail to be included since it does not adhere to `We don’t like install scripts that download unversioned things` so the installation may vary over time depending on released versions of the dependencies. The composer.lock file is there but for some dependencies the requirement is very loose and doesn't specify a specific version ie 5.1.1 instead of 5.x.

**Comment by wturnerharris** [comment]
@javian - I know it used to exist as I contributed to keeping it up to date once or twice. I submitted this new package since I use it, and I prefer using homebrew whenever possible. 

If the potential for dependency nightmares is enough reason to keep it out of the repository, no worries. I could bring up locking down dependency versions with the Pantheon team. Let me know if you have any other ideas.

**Comment by wturnerharris** [comment]
@javian - from one of the terminus maintainers:

> The composer.lock file **does** specify a specific version such as 5.1.1. If composer.json asks for 5.x (or ^5 in Composer parlance), then the specific version selected at the time the composer.lock was created will be recorded and used for all future composer install operations.

It sounds like the lock file is present and utilized during installation, so only the appropriate versions would be installed. The only time the versions in the lock file would change is during a `composer update` operation, which I did not include.

**Pull request closed without merging**